docs/docs/cmd/spo/file/file-retentionlabel-remove.md

```diff
@@ -0,0 +1,43 @@
+# spo file retentionlabel remove
+
+Clears the retention label from a file
+
+## Usage
+
+```sh
+m365 spo file retentionlabel remove [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the retentionlabel from a file to remove is located
+
+`--fileUrl [fileUrl]`
+: The server-relative URL of the file of which the label should be removed. Specify either `fileUrl` or `fileId` but not both.
```

Comment:
This should be a site- or server relative url. We've got a util that you can use to convert that easily: urlUtil.getServerRelativePath